### DISCUSSION TITLE
Deploying Prisma SqLite to Github Pages

### DISCUSSION THREAD

**Discussion by stefensuhat** [prisma/prisma#13150] [Q&A]
Hi, Currently I'm exploring Prisma and really like it. 

Currently I deploy it to Vercel and works flawless. 

My question is it possible to run on github pages?  I also tried deploy it on Vercel -- but without running any Prisma command (so I push the db directly to git), but no luck. I keep getting error. seems like it cannot found the db

Thanks

**Comment by nurul3101** [comment]
Hey @stefensuhat 👋 

GitHub Pages only allow static content so I don't think running SQLite on GitHub pages is officially supported.
However, I was able to find a blog that claimed they were able to use SQLite on GitHub pages [here](https://phiresky.github.io/blog/2021/hosting-sqlite-databases-on-github-pages/). You might want to give that a try.

**Reply by stefensuhat** [reply]
thanks @nurul3101 ! will check it out

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
